Document/0x06f-Testing-Local-Authentication.md

```diff
@@ -225,7 +229,8 @@ If vulnerable, the module will automatically bypass the login form.
 
 #### OWASP MASVS
 
-- V4.7: "Biometric authentication, if any, is not event-bound (i.e. using an API that simply returns "true" or "false"). Instead, it is based on unlocking the keychain/keystore."
+- V4.8: "Biometric authentication, if any, is not event-bound (i.e. using an API that simply returns "true" or "false"). Instead, it is based on unlocking the keychain/keystore."
+- 2.11: "The app enforces a minimum device-access-security policy, such as requiring the user to set a device passcode."
```

Comment:
somehow the requirements are bold?

Reply:
this is because of the __bridge part. now escaped it looks better i guess...

Reply:
Should be fixed now, was basically due to the usage of __

Reply:
So using objc instead of objective-c fixed it in the end... it ws a parser issue